pkg/network/additional_networks_test.go

```diff
@@ -60,3 +67,49 @@ func TestValidateRaw(t *testing.T) {
 	rawConfig.Name = ""
 	errExpect("Additional Network Name cannot be nil")
 }
+
+func TestRenderOpenShiftSRIOV(t *testing.T) {
+	g := NewGomegaWithT(t)
+
+	for _, cfg := range NetworkAttachmentConfig.Spec.AdditionalNetworks {
+		switch cfg.Type {
+		case operv1.NetworkTypeOpenShiftSRIOV:
+			objs, err := renderOpenShiftSRIOV(&cfg, manifestDir)
+			g.Expect(err).NotTo(HaveOccurred())
+			g.Expect(objs).To(HaveLen(6))
+			g.Expect(objs).To(
+				ContainElement(HaveKubernetesID(
+					"NetworkAttachmentDefinition", "default", cfg.Name)))
+		}
+	}
+}
+
+func TestValidateOpenShiftSRIOV(t *testing.T) {
+	g := NewGomegaWithT(t)
+
+	for _, cfg := range NetworkAttachmentConfig.Spec.AdditionalNetworks {
+		switch cfg.Type {
+		case operv1.NetworkTypeOpenShiftSRIOV:
+			err := validateOpenShiftSRIOV(&cfg)
+			g.Expect(err).To(BeEmpty())
+		}
+	}
+
```

Comment:
You should do a `NetworkAttachmentConfig.DeepCopy()` here, and then mutate it locally for the test.

Reply:
Done